Doc/library/os.rst

```diff
@@ -305,7 +305,7 @@ process and user.
 .. function:: getenv(key, default=None)
 
    Return the value of the environment variable *key* if it exists, or
-   *default* if it doesn't. *key*, *default* and the result are str. Note that
+   *default* if it doesn't. *key* and the result are strings. Note that
```

Comment:
The result isn't a string if default isn't one. I'd suggest "Return the value of the environment variable *key* as a string if it exists, or *default* if it doesn't. *key* is a string."